JoinColony/colonyCDapp
# Feat: Update the Colony Switcher UI

## Description

This PR updates the UI of the Colony Switcher.

![Screenshot 2024-08-16 at 18 19 12](https://github.com/user-attachments/assets/6f36bb05-989c-4515-9f2a-4a35770d6f34)

![Screenshot 2024-08-16 at 18 19 27](https://github.com/user-attachments/assets/2bc1c712-9644-4513-81a9-4c4ef33408d2)

## Testing

> [!NOTE]
> The Create new colony button won't work. It will be addressed in a different PR.

1. Connect Dev Wallet 1 so you can view Colonies on the Colony Switcher
2. Verify the UI matches the Figma designs
3. Set the browser width to 1024px
4. Verify the UI matches the Figma designs

Resolves #2885
